recipe/meta.yaml

```diff
@@ -1,17 +1,20 @@
 {% set version = "0.6" %}
-{% set sha256 = "b4be2b3438971588dc306116658f475916755ad8fcc0618e79023846ce94f97b" %}
 
 package:
   name: python-graphviz
   version: {{ version }}
 
 source:
-  fn: python-graphviz-{{ version }}.zip
-  url: https://pypi.io/packages/source/g/graphviz/graphviz-{{ version }}.zip
-  sha256: {{ sha256 }}
+  fn: python-graphviz-{{ version }}.tar.gz
+  url: https://github.com/xflr6/graphviz/archive/{{ version }}.tar.gz
```

Comment:
Can we please change this back to PyPI? GitHub archives, which this uses, unfortunately can change checksums over time. This results in unnecessary breakage, maintenance fixing, and questions as to what might have caused it. As PyPI guarantees the checksum, this is easier from a maintenance side.